DataFormats/PortableTestObjects/src/alpaka/classes_serial.h

```diff
@@ -5,11 +5,12 @@
 #include "DataFormats/Portable/interface/PortableCollectionCommon.h"
 
 namespace ROOT::Meta::Selection {
-  namespace portablecollection {
-    class c0 : CollectionImpl<0, void> {};
+  namespace PortableTestObjects {
```

Comment:
the namespace should be lowercase (sorry for being pedantic)

Reply:
Sure.